main/main.cpp

```diff
@@ -1933,6 +1933,13 @@ Error Main::setup2(Thread::ID p_main_tid_override) {
 	if (editor || project_manager || cmdline_tool) {
 		EditorPaths::create();
 	}
+	if (EditorPaths::get_singleton()->is_self_contained()) {
```

Comment:
This will crash when running a project. EditorPaths is created right above under conditions - you can only use it under those same conditions.

Reply:
`_sc_` seems to be only used by the editor, so I guess it's not relevant when running the project, so I guess it should be in the same `(editor || project_manager || cmdline_tool)` condition.

Reply:
Sorry about that, this seems obvious in retrospect, it's what I get for trying to finish this feature past midnight.